src/metabase/channel/email/alert.hbs

```diff
@@ -1,60 +0,0 @@
-{{> metabase/channel/email/_dashsub_alert_header.hbs }}
```

Comment:
I'm missing why these templates are being deleted in this PR. Where they accidentally left behind by an earlier one?

Reply:
Oh, I see you left a comment about this, but Github hid it as Outdated 😞